websites/E/EternalNetwork TM/presence.ts

```diff
@@ -158,7 +158,7 @@ presence.on("UpdateData", async () => {
 			if (pathname.startsWith("/")) presenceData.state = ttl;
 
 			if (pathname.includes("/admin")) {
-				presenceData.state = "Adding new incident 😥 !";
+				presenceData.state = "Adding new incident !";
```

Comment:
```suggestion
				presenceData.state = "Adding new incident";
```

Reply:
Exclamation marks are removed from the whole presence.